### PR TITLE
ENG-4479 Remove "BulkManager" functionality (7.1.x)

### PR DESCRIPTION
**Pull request by entgigi**

### CODE REVIEW

**Review by eugeniosant — APPROVED** [review]

File: cms-plugin/src/main/resources/spring/plugins/jacms/apsadmin/content/bulkCommandConfig.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    </bean>
+
+    <bean id="jacmsJoinCategoryBulkCommand" parent="jacmsBaseContentBulkCommand" scope="prototype"
+            class="org.entando.entando.plugins.jacms.apsadmin.content.bulk.commands.JoinCategoryBulkCommand" />
+
+    <bean id="jacmsRemoveCategoryBulkCommand" parent="jacmsBaseContentBulkCommand" scope="prototype"
+            class="org.entando.entando.plugins.jacms.apsadmin.content.bulk.commands.RemoveCategoryBulkCommand" />
+    <!--
+    <bean id="jacmsJoinGroupBulkCommand" parent="jacmsBaseContentBulkCommand" scope="prototype"
+              class="org.entando.entando.plugins.jacms.apsadmin.content.bulk.commands.JoinGroupBulkCommand" >
```
Comment (zonia3000) [inline cms-plugin/src/main/resources/spring/plugins/jacms/apsadmin/content/bulkCommandConfig.xml]: Can we remove these commented lines?

File: cms-plugin/src/main/webapp/WEB-INF/plugins/jacms/apsadmin/jsp/content/bulk/bulk-result.jsp [hunk 2]
```diff
@@ -2,98 +2,88 @@
...
-                </s:text>
-            </p>
-            <p>
-                <strong><s:text name="label.bulk.report.successCount" /></strong>: <s:property value="%{#report.applySuccesses}" />
-            </p>
+    <div class="alert alert-success mt-20">
+        <span class="pficon pficon-ok"></span>
+        <%--
+        <p>
+            <strong><s:text name="label.bulk.status" /></strong>: <s:text name="name.bulk.status.%{#command.status}" />
```
Comment (zonia3000) [inline cms-plugin/src/main/webapp/WEB-INF/plugins/jacms/apsadmin/jsp/content/bulk/bulk-result.jsp]: Can we remove these commented lines?

File: cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java [hunk 3]
```diff
@@ -45,87 +37,97 @@
...
 		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
-		
-		result = this.executeCheckResult(currentUser, null);
-		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
-		
-		result = this.executeViewResult(currentUser, null);
+
+		result = this.executeCheckResult(currentUser);
 		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
+// FIXME
```
Comment (zonia3000) [inline cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java]: ?

File: cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java [hunk 4]
```diff
@@ -137,24 +139,14 @@ void testApplyAddRemove() throws Throwable {
...
-			result = this.executeViewResult(currentUser, commandId);
-			assertEquals(Action.SUCCESS, result);
-
-			commandId = ((ContentBulkAction) this.getAction()).getCommandId();
-			this.checkReport(commandId, size, size, size, 0, ApsCommandStatus.COMPLETED);
 
+//			result = this.executeCheckResult(currentUser);
+//			assertEquals(Action.SUCCESS, result);
+//			result = this.executeViewResult(currentUser);
+//			assertEquals(Action.SUCCESS, result);
```
Comment (zonia3000) [inline cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java]: Can we remove these commented lines?

File: cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java [hunk 5]
```diff
@@ -260,29 +251,15 @@ private String executeGroupAction(String currentUser, String actionName, int str
...
-			assertNotNull(report.getEndingTime());
-		}
-	}
-
-    @BeforeEach
+	@BeforeEach
 	private void init() {
 		this._contentManager = (IContentManager) this.getApplicationContext().getBean(JacmsSystemConstants.CONTENT_MANAGER);
-		this._bulkCommandManager = (IBulkCommandManager) this.getApplicationContext().getBean(SystemConstants.BULK_COMMAND_MANAGER);
+		//this._bulkCommandManager = (IBulkCommandManager) this.getApplicationContext().getBean(SystemConstants.BULK_COMMAND_MANAGER);
```
Comment (zonia3000) [inline cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java:251]: Can we remove these commented lines?

File: cms-plugin/src/main/resources/spring/plugins/jacms/apsadmin/content/bulkCommandConfig.xml (hunk 1 above)
Comment (entgigi) [reply cms-plugin/src/main/resources/spring/plugins/jacms/apsadmin/content/bulkCommandConfig.xml]: there was a comment in the original code in entando-ps, i will remove it

File: cms-plugin/src/main/webapp/WEB-INF/plugins/jacms/apsadmin/jsp/content/bulk/bulk-result.jsp (hunk 2 above)
Comment (entgigi) [reply cms-plugin/src/main/webapp/WEB-INF/plugins/jacms/apsadmin/jsp/content/bulk/bulk-result.jsp]: there was a comment in the original code in entando-ps, i will remove it

File: cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java (hunk 3 above)
Comment (entgigi) [reply cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java]: I put a fixme to talk with @eugeniosant, i will remove it

File: cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java (hunk 4 above)
Comment (entgigi) [reply cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java]: there was a comment in the original code in entando-ps, i will remove it

File: cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java (hunk 5 above)
Comment (entgigi) [reply cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java:251]: there was a comment in the original code in entando-ps, i will remove it

File: cms-plugin/src/main/resources/spring/plugins/jacms/apsadmin/content/bulkCommandConfig.xml (hunk 1 above)
Comment (entgigi) [reply cms-plugin/src/main/resources/spring/plugins/jacms/apsadmin/content/bulkCommandConfig.xml]: done

File: cms-plugin/src/main/webapp/WEB-INF/plugins/jacms/apsadmin/jsp/content/bulk/bulk-result.jsp (hunk 2 above)
Comment (entgigi) [reply cms-plugin/src/main/webapp/WEB-INF/plugins/jacms/apsadmin/jsp/content/bulk/bulk-result.jsp]: done

File: cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java (hunk 4 above)
Comment (entgigi) [reply cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java]: done

File: cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java (hunk 5 above)
Comment (entgigi) [reply cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java:251]: done

**Review by zonia3000 — APPROVED** [review]

**Pull request merged**